### ISSUE TITLE
Get involved page

### ISSUE THREAD

**Issue by melissasamworth** [StampyAI/stampy-ui#374]
**Acceptance criteria**

- The sidebar logic discussed should take care of the sidebar completely (i.e. that is not part of this task)
- The article content itself also matches articles from other sections
- The only thing different is the bottom section, where the "Keep going" section currently appears
  - We don't want the standard "Related questions" to show up at the bottom of the pages, so we'll likely just want to delete Related questions in Coda
  - The bottom, instead, just shows it's children
  - If the article is the third tier (i.e. has no children), obviously this won't show

**Links**
https://www.figma.com/file/qqfeYAh6Z7QdyJXsQjIUfh/Stampy?type=design&node-id=2177%3A4109&mode=dev&t=VPCJqtZ3vZNguLmb-1

**Comment by melissasamworth** [comment]
Blocked by keep going section

**Comment by melissasamworth** [comment]
Update:

There should be one more set of children. Is this a copy thing or a dev thing? 

The little arrows are cut off on some

![Image](https://github.com/StampyAI/stampy-ui/assets/825910/2c915b01-3026-4088-b4a4-0edb4e0ba985)

The article content itself is not right but I think this is just blocked by the "main content on article page" which I hope to fix soon.

**Comment by melissasamworth** [comment]
Same as before, that there should be one more set of children. Is this a copy thing or a dev thing?

**Comment by melissasamworth** [comment]
Oh and actually for this section only, can we not open the children in a new tab? Remove the arrows too. I am sorry my fault.



![Image](https://github.com/StampyAI/stampy-ui/assets/825910/e9245a00-c72b-4c32-ad2d-87eceaa7d063)

**Comment by melissasamworth** [comment]
Update: Melissa talk to editors about putting third tier

**Comment by melissasamworth** [comment]
Looks good enough, but (another task) @mruwnik There are a couple CSS comments I have about the sidebar, to be addressed later
